### PR TITLE
Feat/net apy tooltip

### PR DESCRIPTION
**Pull request by defispartan**
Add explanation for net APY

### CODE REVIEW

**Review by foodaka — APPROVED** [review]

File: src/modules/dashboard/DashboardTopPanel.tsx [hunk 1]
```diff
@@ -107,7 +108,16 @@ export const DashboardTopPanel = () => {
           )}
         </TopInfoPanelItem>
 
-        <TopInfoPanelItem icon={<NetAPYIcon />} title={<Trans>Net APY</Trans>} loading={loading}>
+        <TopInfoPanelItem
+          icon={<NetAPYIcon />}
+          title={
+            <div style={{ display: 'flex' }}>
```
Comment (sakulstra) [inline src/modules/dashboard/DashboardTopPanel.tsx:114]: Not at the PC right now so little hard to check, but I think we have a predefined textwithtooltip component somewhere

Comment (defispartan) [reply src/modules/dashboard/DashboardTopPanel.tsx:114]: That component was for wrapping the actual tooltip text. There was another component which took text and tooltip as props called ListTopInfoItem but that was specific to the dashboard headers

**Comment by defispartan** [comment]
I reworded the tooltip to better explain this scenario:

Screenshot from 2022-03-24 14-35-59

The net APY is relative to *net worth*, so it's possible to have a lower net APY than debt APY if the user is leveraged

File: src/components/infoTooltips/NetAPYTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Trans } from '@lingui/macro';
+
+import { TextWithTooltip, TextWithTooltipProps } from '../TextWithTooltip';
+
+export const NetAPYTooltip = ({ ...rest }: TextWithTooltipProps) => {
+  return (
+    <TextWithTooltip {...rest}>
+      <Trans>
+        The net APY is the combined effect of all supply and borrow positions on net worth,
```
Comment (foodaka) [inline src/components/infoTooltips/NetAPYTooltip.tsx:9]:
```suggestion
       Net APY is the combined effect of all supply and borrow positions on net worth,
```

**Review by foodaka — APPROVED** [review]

**Pull request merged**